test/e2e/network/dns_common.go

```diff
@@ -406,10 +406,10 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 		}
 		fileName := fmt.Sprintf("%s_udp@%s", fileNamePrefix, name)
 		fileNames = append(fileNames, fileName)
-		probeCmd += fmt.Sprintf(`test -n "$$(dig +notcp +noall +answer +search %s %s)" && echo OK > /results/%s;`, name, lookup, fileName)
+		probeCmd += fmt.Sprintf(`test -n "$$(dig +notcp +noall +answer +search %s %s | grep -v 'connection timed out')" && echo OK > /results/%s;`, name, lookup, fileName)
```

Comment:
looks like we are here dependent of the error msg received, and on which protocol.

Would'nt be better to verify the output of the dig command, showing we have a kind of response.
even if that is only part of the labels ?
That would be more sure to avoid the other kind of errors.

if I test "

`dig +notcp +noall +answer +search www.google.com A`

I get

`www.google.com.		151	IN	A	216.58.194.164`

should we test, either "IN", either domain back ... ?

Reply:
Maybe return code is cleaner to check for the "no reply from server" case...

```
Dig return codes are:

0: Everything went well, including things like NXDOMAIN
1: Usage error
8: Couldn't open batch file
9: No reply from server
10: Internal error
```